### ISSUE TITLE
IDISP006 should not warn when overriding Dispose(disposing)

### ISSUE THREAD

**Issue by JohanLarsson** [DotNetAnalyzers/IDisposableAnalyzers#97]
```cs
        [Test]
        public void WhenCallingBaseDispose()
        {
            var baseCode = @"
namespace RoslynSandbox
{
    using System;

    public abstract class FooBase : IDisposable
    {
        private readonly IDisposable disposable = new Disposable();
        private bool disposed;

        /// <inheritdoc/>
        public void Dispose()
        {
            this.Dispose(true);
        }

        protected virtual void Dispose(bool disposing)
        {
            if (this.disposed)
            {
                return;
            }

            this.disposed = true;
            if (disposing)
            {
                this.disposable.Dispose();
            }
        }
    }
}";
            var testCode = @"
namespace RoslynSandbox
{
    using System;

    public class Foo : FooBase
    {
        private readonly IDisposable disposable = new Disposable();
        private bool disposed;

        protected override void Dispose(bool disposing)
        {
            base.Dispose(disposing);
        }
    }
}";

            AnalyzerAssert.Valid(Analyzer, ExpectedDiagnostic, DisposableCode, baseCode, testCode);
        }
```
